.github/workflows/profanity-filter.yml

```diff
@@ -0,0 +1,26 @@
+name: Profanity filter
+
+on:
+  issue_comment:
```

Comment:
Does `issue_comment` include PR comment / review comment? If not, this is worth adding.

Reply:
Review comments have not been considered, as we're usually not the one's using profanity. But that might be worth adding support for. And for `issue_comment`, that covers `pull_request_comment`, see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_comment-use-issue_comment.